test/e2e/resources/pull_through_cache_rule.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: ecr.services.k8s.aws/v1alpha1
+kind: PullThroughCacheRule
+metadata:
+  name: $NAME
+spec:
+  registryID: "771174509839"
```

Comment:
Any information about this number? Would you mind at least leaving a comment to explain it?

Reply:
Haaaa... nice catch. This is why e2e are failing on prow but passing with my account...